### ISSUE TITLE
season 1 

### ISSUE THREAD

**Issue by Torstentjh** [Rakha112/react-native-animation#4]
hello，recently i try to use r3f ，I encountered some problems and saw the issue you mentioned earlier in the official repository, but I still couldn't solve it. I tried both gltf and glb formats, and even copied your code in 17 completely in my project, but the simulator still didn't render anything. When I used the gltd format, it also warned error:THREE.GLTFLoader: Couldn’t load texture .....
so I want to ask if you have ever encountered this situation.

**Comment by Rakha112** [comment]
Hey @Torstentjh, are you using the 3D Model from this repo or using your own 3D Model ?

**Comment by Torstentjh** [comment]
> Hey @Torstentjh, are you using the 3D Model from this repo or using your own 3D Model ?

I didn't directly clone your repository and then run the project,  I copied the glb file and code to my project  ，I have tried both the ones in your repository or the ones I downloaded, and none of them completed. Then the gltf format will report an error, and the glb format has no error but does not render anything

like this 
![image](https://github.com/Rakha112/react-native-animation/assets/75771172/a0913d84-27aa-4a0a-85c6-422f4ad3c393)

**Comment by Rakha112** [comment]
can you tell me the versions of React Native, Expo, React Three Fiber, and React Three Drei that you're using ?

**Comment by Torstentjh** [comment]
i use cli init project，not expo，I didn't pay attention to the version issue, so I downloaded the new version directly
![image](https://github.com/Rakha112/react-native-animation/assets/75771172/662278ab-b6a7-402e-ac96-15dd1f953c1f)

**Comment by Rakha112** [comment]
Yes, I think with the latest version there should be no problem, regarding that error, I have encountered it before with an older version of React Three Fiber https://github.com/pmndrs/react-three-fiber/issues/3075, but I am sure the error has been fixed for the latest version, can you please create a reproducible repository that I can clone and test ?

**Comment by Torstentjh** [comment]
sorry，I just tried to create a new project, download those dependencies and run the project. Unfortunately, I have network problems here, the download is very slow, and I encountered other problems, which are very troublesome.🥲

**Comment by Rakha112** [comment]
Maybe you can use a VPN. I've experienced this before and used WARP from Cloudflare, then the download go back to normal

**Comment by Torstentjh** [comment]
It crashed and still didn't run the new project. There were many errors, gradle version and many other problems. I solved this problem temporarily. Sorry to bother you. Thank you.

**Comment by Infiee** [comment]
Thank you for sharing so many animation effects. You are really meticulous and patient. Thank you.

**Comment by denysoleksiienko** [comment]
@Rakha112 Thank you for sharing your code. I have a question about Accordion. Is it hard to implement close previously opened item in nested accordion If open next?

**Comment by rthic23** [comment]
@Rakha112 do you have code for bottomsheet using Modal component which has snappoints?
